src/main/java/com/kodilla/tictactoe/Combo.java

```diff
@@ -0,0 +1,22 @@
+package com.kodilla.tictactoe;
+
+public class Combo {
+
+    private Tile[] tiles;
+
+    public Combo(Tile... tiles) {
+        this.tiles = tiles;
+    }
+
+    public boolean isComplete() {
+        if (!(tiles[0].getValue() == Game.type.O) && !(tiles[0].getValue() == Game.type.X))
```

Comment:
używaj nawiasów klamrowych ` { } `

```
if (!(tiles[0].getValue() == Game.type.O) && !(tiles[0].getValue() == Game.type.X)) {
    return false;
}
return tiles[0].getValue().equals(tiles[1].getValue())
                && tiles[0].getValue().equals(tiles[2].getValue());
```

Reply:
W niektórych przykładach z internetu jest, że jak jest tylko jedna komenta po if to nie ma klamerek. To tylko dla estetyki czy to ma jakieś znaczenie przy kompilowaniu ?

Reply:
nie ma znaczenia dla kompiltora, po prostu polepsza to czytelność.